ui/src/cpu_handler.rs

```diff
@@ -118,89 +173,128 @@ impl UiTool for CpuHandler {
-                if let Ok(mut gba) = self.gba.lock() {
-                    gba.step()
+            ui.horizontal(|ui| {
+                ui.label("Skip CPU cycle:");
+
+                if ui.button("⏭x1").clicked() {
+                    if let Ok(mut gba) = self.gba.lock() {
+                        gba.step()
+                    }
                 }
-            }
 
-            if ui.button("⏭x10").clicked() {
-                if let Ok(mut gba) = self.gba.lock() {
-                    (0..10).for_each(|_| gba.step());
+                if ui.button("⏭x10").clicked() {
+                    if let Ok(mut gba) = self.gba.lock() {
+                        (0..10).for_each(|_| gba.step());
+                    }
                 }
-            }
 
-            if ui.button("⏭x100").clicked() {
-                if let Ok(mut gba) = self.gba.lock() {
-                    (0..100).for_each(|_| gba.step());
+                if ui.button("⏭x100").clicked() {
+                    if let Ok(mut gba) = self.gba.lock() {
+                        (0..100).for_each(|_| gba.step());
+                    }
                 }
-            }
 
-            if ui.button("⏭x500").clicked() {
-                if let Ok(mut gba) = self.gba.lock() {
-                    (0..500).for_each(|_| gba.step());
+                if ui.button("⏭x500").clicked() {
+                    if let Ok(mut gba) = self.gba.lock() {
+                        (0..500).for_each(|_| gba.step());
+                    }
                 }
-            }
-        });
 
-        ui.add_space(20.0);
+                if ui.button("⏭x1000").clicked() {
+                    if let Ok(mut gba) = self.gba.lock() {
+                        (0..1000).for_each(|_| gba.step());
+                    }
+                }
+            });
 
-        ui.heading("Breakpoints");
-        ui.add_space(12.0);
+            ui.horizontal(|ui| {
+                ui.label("Custom CPU cycle to skip:");
+                ui.add(egui::DragValue::new(&mut self.cycle_to_skip_custom_value).speed(100));
 
-        ui.horizontal(|ui| {
-            ui.label("Address (hex) : ");
-            ui.text_edit_singleline(&mut self.b_address);
-
-            egui::ComboBox::from_label("Select breakpoint type")
-                .selected_text(if self.breakpoint_combo == BreakpointType::Equal {
-                    "="
-                } else {
-                    ">"
-                })
-                .show_ui(ui, |ui| {
-                    ui.style_mut().wrap = Some(false);
-                    ui.set_min_width(60.0);
-                    ui.selectable_value(&mut self.breakpoint_combo, BreakpointType::Equal, "=");
-                    ui.selectable_value(&mut self.breakpoint_combo, BreakpointType::Greater, ">");
-                });
-
-            if ui.button("Add").clicked() {
-                if self.b_address.is_empty() {
-                    return;
+                if ui.button("Skip").clicked() {
```

Comment:
I would say "exec" or "exec in batch" or just batch
With skip i think those are not executed

Reply:
What do you think about `execute and stop` or `step`?

Reply:
Step is good and short :)